### ISSUE TITLE
Fails to compile when installed with hyprpm

### ISSUE THREAD

**Issue by LeSnake04** [Duckonaut/split-monitor-workspaces#57]
```
FAILED: libsplit-monitor-workspaces.so.p/src_main.cpp.o
sccache c++ -Ilibsplit-monitor-workspaces.so.p -I. -I.. -I../include -I/home/lesnake/.local/share/hyprpm/headersRoot/include/hyprland/protocols -I/home/lesnake/.local/share/hyprpm/headersRoot
/include/hyprland/wlroots -I/home/lesnake/.local/share/hyprpm/headersRoot/include -I/usr/include/pixman-1 -I/usr/include/libdrm -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winval
id-pch -O3 -std=c++23 -DWLR_USE_UNSTABLE -fPIC -MD -MQ libsplit-monitor-workspaces.so.p/src_main.cpp.o -MF libsplit-monitor-workspaces.so.p/src_main.cpp.o.d -o libsplit-monitor-workspaces.so.
p/src_main.cpp.o -c ../src/main.cpp
../src/main.cpp: In function ‘void mapWorkspacesToMonitors()’:
../src/main.cpp:125:45: error: ‘Hyprlang’ has not been declared
  125 |     static auto* const workspaceCountPtr = (Hyprlang::INT* const*)HyprlandAPI::getConfigValue(PHANDLE, k_workspaceCount)->getDataStaticPtr();
      |                                             ^~~~~~~~
../src/main.cpp:125:60: error: expected primary-expression before ‘const’
  125 |     static auto* const workspaceCountPtr = (Hyprlang::INT* const*)HyprlandAPI::getConfigValue(PHANDLE, k_workspaceCount)->getDataStaticPtr();
      |                                                            ^~~~~
../src/main.cpp:125:59: error: expected ‘)’ before ‘const’
  125 |     static auto* const workspaceCountPtr = (Hyprlang::INT* const*)HyprlandAPI::getConfigValue(PHANDLE, k_workspaceCount)->getDataStaticPtr();
      |                                            ~              ^~~~~~
      |                                                           )
../src/main.cpp:126:42: error: ‘Hyprlang’ has not been declared
  126 |     static auto* const keepFocusedPtr = (Hyprlang::INT* const*)HyprlandAPI::getConfigValue(PHANDLE, k_keepFocused)->getDataStaticPtr();
      |                                          ^~~~~~~~
../src/main.cpp:126:57: error: expected primary-expression before ‘const’
  126 |     static auto* const keepFocusedPtr = (Hyprlang::INT* const*)HyprlandAPI::getConfigValue(PHANDLE, k_keepFocused)->getDataStaticPtr();
      |                                                         ^~~~~
../src/main.cpp:126:56: error: expected ‘)’ before ‘const’
  126 |     static auto* const keepFocusedPtr = (Hyprlang::INT* const*)HyprlandAPI::getConfigValue(PHANDLE, k_keepFocused)->getDataStaticPtr();
      |                                         ~              ^~~~~~
      |                                                        )
../src/main.cpp: In function ‘PLUGIN_DESCRIPTION_INFO pluginInit(void*)’:
../src/main.cpp:170:60: error: ‘Hyprlang’ has not been declared
  170 |     HyprlandAPI::addConfigValue(PHANDLE, k_workspaceCount, Hyprlang::INT{10});
      |                                                            ^~~~~~~~
../src/main.cpp:171:57: error: ‘Hyprlang’ has not been declared
  171 |     HyprlandAPI::addConfigValue(PHANDLE, k_keepFocused, Hyprlang::INT{0});
      |                                                         ^~~~~~~~
ninja: build stopped: subcommand failed.
```

**Comment by bnema** [comment]
Same issue here

**Comment by rickalex21** [comment]
Same here, I'm not too familiar with cpp but it looks like it could be a coding error?
I'm not sure if others are experiencing the same error about `multiple definition`.

```text
Found ninja-1.11.1 at /usr/bin/ninja
                                                
 -> cd /tmp/hyprpm/new && PKG_CONFIG_PATH="/home/rick/.local/share/hyprpm/headersRoot/share/pkgconfig" meson compile -C build
ninja: Entering directory `/tmp/hyprpm/new/build'
[1/2] Compiling C++ object libsplit-monitor-workspaces.so.p/src_main.cpp.o
FAILED: libsplit-monitor-workspaces.so.p/src_main.cpp.o 
c++ -Ilibsplit-monitor-workspaces.so.p -I. -I.. -I../include -I/home/rick/.local/share/hyprpm/headersRoot/include/hyprland/protocols -I/home/rick/.local/share/hyprpm/headersRoot/include/hyprland/wlroots -I/home/rick/.local/share/hyprpm/headersRoot/include -I/usr/include/pixman-1 -I/usr/include/libdrm -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -O3 -std=c++23 -DWLR_USE_UNSTABLE -fPIC -MD -MQ libsplit-monitor-workspaces.so.p/src_main.cpp.o -MF libsplit-monitor-workspaces.so.p/src_main.cpp.o.d -o libsplit-monitor-workspaces.so.p/src_main.cpp.o -c ../src/main.cpp
In file included from ../src/main.cpp:6:
/home/rick/.local/share/hyprpm/headersRoot/include/hyprland/src/helpers/Workspace.hpp:7:6: error: multiple definition of ‘enum eFullscreenMode’
    7 | enum eFullscreenMode : int8_t {
      |      ^~~~~~~~~~~~~~~
In file included from /home/rick/.local/share/hyprpm/headersRoot/include/hyprland/src/Compositor.hpp:24,
                 from ../src/main.cpp:3:
/home/rick/.local/share/hyprpm/headersRoot/include/hyprland/src/desktop/Workspace.hpp:7:6: note: previous definition here
    7 | enum eFullscreenMode : int8_t {
      |      ^~~~~~~~~~~~~~~
/home/rick/.local/share/hyprpm/headersRoot/include/hyprland/src/helpers/Workspace.hpp:15:7: error: redefinition of ‘class CWorkspace’
   15 | class CWorkspace {
      |       ^~~~~~~~~~
/home/rick/.local/share/hyprpm/headersRoot/include/hyprland/src/desktop/Workspace.hpp:15:7: note: previous definition of ‘class CWorkspace’
   15 | class CWorkspace {
      |       ^~~~~~~~~~
In file included from /home/rick/.local/share/hyprpm/headersRoot/include/hyprland/src/Compositor.hpp:15:
/home/rick/.local/share/hyprpm/headersRoot/include/hyprland/src/managers/KeybindManager.hpp:184:41: warning: ‘void toggleActiveFloatingCore(std::string, std::optional<bool>)’ declared ‘static’ but never defined [-Wunused-function]
  184 | static void                             toggleActiveFloatingCore(std::string args, std::optional<bool> floatState);
      |                                         ^~~~~~~~~~~~~~~~~~~~~~~~
ninja: build stopped: subcommand failed.
INFO: autodetecting backend as ninja
INFO: calculating backend command to run: /usr/bin/ninja -C /tmp/hyprpm/new/build


✖ Plugin split-monitor-workspaces failed to build.                                                                          

✔ all plugins built                                                                                                         
✔ installed repository                                                                                                      
✔ you can now enable the plugin(s) with hyprpm enable                                                                       
 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━  5 / 5  Done!       
```

**Comment by Duckonaut** [comment]
Fixed in #52
